src/app/routes/assets/asset-ownership-history/asset-ownership-history.component.html

```diff
@@ -8,12 +8,15 @@ <h4 mat-dialog-title class="mb-0 d-flex justify-content-between align-items-cent
 <div mat-dialog-content>
   <!-- Asset Ownership list Layout -->
   <div class="history-list py-2">
-    <div class="history-list-item my-3 px-2 pb-3" *ngFor="let element of dataSource.data">
+    <div
+      class="history-list-item my-3 px-2 pb-3"
+      [attr.data-qa-id]="'history-item-' + id"
+      *ngFor="let element of dataSource.data; let id = index">
       <div class="d-flex justify-content-between align-items-center pt-3 px-2">
-        <h6 class="font-weight-600">
-          {{ element?.emittedDate | date: 'MM/dd/yyyy HH:mm:ss' }}
+        <h6 class="font-weight-600" data-qa-id="emitted-date">
+          {{ element?.emittedDate | defaultDate }}
```

Comment:
Where does `defaultDate` come from here? Is `defaultDate` equivalent to what was here before (`date: 'MM/dd/yyyy HH:mm:ss'`)?

Reply:
It's from here: https://github.com/energywebfoundation/switchboard-dapp/blob/develop/src/app/shared/pipes/default-date/default-date.pipe.ts
and yes, it's equivalent. Just a wrapper if we would like to change default format to a different one, then we change it in one place, not in entire app.

Reply:
Sounds great, thanks